src/vellum/workflows/workflows/base.py

```diff
@@ -86,8 +86,11 @@ def __new__(mcs, name: str, bases: Tuple[Type, ...], dct: Dict[str, Any]) -> Any
         return super().__new__(mcs, name, bases, dct)
 
 
+GraphAttribute = Union[Type[BaseNode], Graph, Set[Type[BaseNode]], Set[Graph]]
```

Comment:
This is ultimately why I called the generator `graph-attribute` instead of `graph`. `Graph` is a specific class in the SDK, and we are ultimately looking to generate this union